src/Exceptions/InvalidResourceId.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Craftzing\Laravel\MollieWebhooks\Exceptions;
+
+use Exception;
+
+use function sprintf;
+
+final class InvalidResourceId extends Exception
+{
+    public static function missingExpectedPrefix(string $value, string $prefix): self
+    {
+        return new self(
+            "Value `$value` cannot be used as a Mollie resource identifier as it " .
+            sprintf("is missing the expected `%s` prefix.", $prefix),
```

Comment:
I may have used `sprintf` in the original exception as well, but now that I look back to it, it seems pretty redundant 🙈 I'd just go with `"is missing the expected `$prefix` prefix."`

Reply:
Haha, I was wondering about this because we had a conversation about it some time ago.